docs/network/smart-rollup-nodes.md

```diff
@@ -57,144 +47,134 @@ The best place to get the most recent binary files to use with Etherlink is http
+   octez-smart-rollup-node --endpoint https://rpc.tzkt.io/mainnet run \
+     --data-dir $SR_DATA_DIR
    ```
 
    If you did not load a snapshot, the process of starting the node from genesis can take a long time because it must process every block.
 
+   If you loaded a snapshot, the node must process every block that has been created since the snapshot was taken, which takes time depending on the age of the snapshot.
+
 1. Verify that the Smart Rollup node is running by querying it.
 For example, this query gets the health of the node:
 
    ```bash
    curl -s http://localhost:8932/health
    ```
 
-1. Ensure that the node runs persistently.
-Look up how to run programs persistently in the documentation for your operating system.
+Now the Smart Rollup node is running and tracking the state of Etherlink by receiving information from layer 1.
 
-1. If you started the node with a public layer 1 node, stop it and restart it with a layer 1 node that you control.
+## Converting the Smart Rollup node to maintenance mode
 
-Now that you have a Smart Rollup node configured for Etherlink, you can run an Etherlink EVM node, as described in [Running an Etherlink EVM node](./evm-nodes).
+Converting the observer node to maintenance mode requires these prerequisites:
 
-## Running the Smart Rollup node in observer mode
-
-Running the Smart Rollup node in observer mode is the simplest way to run an Etherlink Smart Rollup node, but it does not strengthen the security of the system by posting commitments.
-In observer mode, the node only monitors the state of Etherlink.
-
-1. Get a built version of the Smart Rollup node binary, named `octez-smart-rollup-node`.
-The best place to get the most recent binary files to use with Etherlink is https://gitlab.com/tezos/tezos/-/releases.
+- Your own Tezos layer 1 node running in any mode.
 
-1. Initialize the local context of the node, which is where it stores local data:
+   :::warning Use a private layer 1 node
 
-   1. Set the environment variable `SR_DATA_DIR` to the directory where the node should store its local data.
-   The default value is `$HOME/.tezos-smart-rollup-node`.
+   To run the Smart Rollup node in a mode that posts commitments, including maintenance mode and operator mode, you must connect it to a layer 1 node that you control.
+   Using a public layer 1 node as the basis for a Smart Rollup node in maintenance or operator mode exposes the Smart Rollup node to security risks.
+   For example, a malicious layer 1 node can expose the Smart Rollup node to an incorrect branch, which can cause the Smart Rollup node to post invalid commitments and lose tez when other node operators computing from the correct branch refute them.
 
-   1. Initialize the local context by running this command and using the address of the Etherlink Smart Rollup, which you can get from the [Network information](/get-started/network-information) page:
```

Comment:
I would advise to (1) bootstrap on a public node if required on observer mode, (2) switch to your own node still in observer mode, (3) assess the rollup node keep operating as expected, then (4) switch to maintenance mode.

Reply:
Got it, reworked the order to switch to your node first and then go to maintenance mode.